### ISSUE TITLE
File Limit Request: whacc - 120 MB

### ISSUE THREAD

**Issue by PhillipMaire** [pypa/pypi-support#2359]
### Project URL

https://pypi.org/project/whacc/

### Does this project already exist?

- [X] Yes

### New Limit

120

### Update issue title

- [X] I have updated the title.

### Which indexes

PyPI

### About the project

Mouse and rat whiskers are used a model systems in neuroscience/behavioral research in a few dozen labs around the world. My package **Wh**isker **A**utomated **C**ontact **C**lassifier (WhACC) is a complete package to process high speed videos using a CNN+LGBM hybrid model that I have been working on for 2.5 years. it includes GUIs and other user interfaces to solve a tedious task specify to labs that study the whisker system, touch onset and offset. It includes both a ResNetV2 model and Light Gradient Boosted Model that take up some space. Recently, my package after compiled has been right around 104 mb and was uploaded without an issue but now it is 107 mb and I can't upload it anymore. I also include a 32.9mb file that is needed to reference feature names and positions (for plotting figures for the paper which must be self contained) 

### Reasons for the request

I only need a few more MB so that my package can be all inclusive with my training data (needed by the end user) along with data needed for plotting for my final publication of my paper. having everything in one place also looks professional when I am presenting this to get jobs in the future. 

I don't expect this project to increase in size in the future since I am the sole maintainer and coder and because once it is finished I don't intend to update it unless its needs minor fixes for future versions of python or dependent packages. 

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct
